pkg/interface/src/views/apps/launch/components/tiles/basic.js

```diff
@@ -1,55 +1,24 @@
 import React  from 'react';
 import classnames from 'classnames';
-import { Link } from 'react-router-dom';
-import defaultApps from '~/logic/lib/default-apps';
+import { Text, Icon } from '@tlon/indigo-react';
 
 import Tile from './tile';
 
 export default class BasicTile extends React.PureComponent {
   render() {
     const { props } = this;
 
-    const children = (
-      <span>
-        <p className={
-           classnames('absolute f9',
-           { 'black white-d': props.title !== 'Dojo',
-             'white': props.title === 'Dojo' })}
-           style={{ left: 8, top: 8 }}
-        >
-          {props.title}
-        </p>
-         <img
-           className={classnames('absolute',
-           { 'invert-d': props.title !== 'Dojo' })}
-           style={{ left: 38, top: 38 }}
-           src={props.iconUrl}
-           width={48}
-           height={48}
-         />
-      </span>
-    );
-
-    const routeList = defaultApps.map((e) => {
-      return `/~${e}`;
-    });
-
-    const tile =  ( routeList.indexOf(props.linkedUrl) !== -1 ) ? (
-      <Link className="w-100 h-100 db pa2 no-underline" to={props.linkedUrl}>
-       {children}
-      </Link>
-    ) : (
-      <a className="w-100 h-100 db pa2 no-underline" href={props.linkedUrl}>
-       {children}
-      </a>
-    );
-
     return (
-      <Tile>
-        <div className={classnames('w-100 h-100 relative ba b--gray3 b--gray2-d bg-gray0-d br2',
-        { 'bg-white': props.title !== 'Dojo',
-          'bg-black': props.title === 'Dojo' })}
-        >{tile}</div>
+      <Tile
+        bg={props.title === 'Dojo' ? 'black' : 'white'}
```

Comment:
This reverses in dark mode when you really just want it to always be black regardless I think. I think you can just pass a hex color instead of using the theme's keywords in that case

<img width="218" alt="image" src="https://user-images.githubusercontent.com/20846414/95253077-e116aa00-07eb-11eb-94f2-f45edbb2011b.png">